frontend/packages/console-shared/src/components/multi-tab-list/MultiTabListPage.tsx

```diff
@@ -47,6 +46,18 @@ const MultiTabListPage: React.FC<MultiTabListPageProps> = ({
     }
   };
 
+  const items = Object.keys(menuActions).reduce((acc, key) => {
+    const menuAction: MenuAction = menuActions[key];
+    const label =
+      menuAction.label || menuAction.model?.labelKey ? t(menuAction.model.labelKey) : null;
```

Comment:
What we've done elsewhere is include a default to the menuAction.model.label if there's no labelKey.

Reply:
I'll add the fallback. Thanks for the suggestion!